packages/ag-charts-community/src/chart/series/cartesian/rangeBarOptions.ts

```diff
@@ -75,7 +75,9 @@ export type AgRangeBarSeriesLabelPlacement = 'inside' | 'outside';
 /** Configuration for RangeBar series. */
 export interface AgRangeBarSeriesOptions<DatumType = any> extends AgBaseSeriesOptions<DatumType> {
     /** Configuration for the RangeBar series. */
-    type?: 'range-bar' | 'range-column';
+    type: 'range';
```

Comment:
should this be `range-bar` as we have type `range-area`?

Reply:
Fixe